translation/en/RedirectedResources/assets/abdata/h/list/00_00/personality_voice_c00_03_00/translation.txt

```diff
@@ -820,10 +820,10 @@
 //お尻持ってかれちゃう=
 //気持ち良くてお尻が締まっちゃう=
 //乱暴でいいの、そのまま続けてぇっ=
-//お腹の奥がジンジンするわ=
+//お腹の奥がジンジンするわ=I feel it tingling and rubbing inside my belly~
```

Comment:
Indeed, tingling sounds weird.

What I'm still not sure about is ジンジン, I understood that it's "tingling sensation" and because する "to rub" is used afterwards I thought about including both words to emphasize the description.

Do you think there could be any word or expression that translates well ジンジン? I would like to keep the emphasis of the line if it's possible.

Reply:
My mistake again, 
する doesn't mean "to rub" but "to do". We can translate ジンジン as "throbbing".
Putting everything together, "The back of my belly is throbbing."

Reply:
Sounds great, I'm changing it to "The back of my belly is throbbing.".